src/primitive_crypto.cc

```diff
@@ -24,6 +24,44 @@
 
 namespace toit {
 
+enum class AesTypes {
+  Ecb,
+  Cbc
+};
+
+static Object* aes_crypt(Process* process, AesContext* context, Blob* input, bool encrypt, AesTypes type) {
```

Comment:
This has almost no shared code, and I would prefer two functions, getting rid of the static_cast and the enum.

Reply:
Moved the shared method into the two primitives. Removed the enum.